liblwgeom/lwgeom_geos.h

```diff
@@ -38,6 +38,7 @@
 */
 LWGEOM *GEOS2LWGEOM(const GEOSGeometry *geom, char want3d);
 GEOSGeometry * LWGEOM2GEOS(const LWGEOM *g, int autofix);
+GEOSGeometry * LWGEOM2GEOS_PREC(const LWGEOM *g, int autofix, const double* precision);
```

Comment:
please document the semantic of the new parameter above the signature in header file (here)

Reply:
specify (after double-checking) that the geometry coordinates would possibly _move_ due to snapping to the precision grid.